### DISCUSSION TITLE
Documentation

### DISCUSSION THREAD

**Discussion by davidroman0O** [ergo-services/ergo#147] [Q&A] [answered]
Hello!

A simple question: is there any ETA about [the documentation](https://docs.ergo.services/)? 😄 

I know you're working on it!

We're not all coming with an Elixir background and it would help to understand all the concepts so people can help to grow the amount of repositories that use `ergo`!

**Comment by halturin** [comment, marked as answer]
will be released along with the 3.0. scheduled this fall.
